### ISSUE TITLE
Blog post: sounding the depths of possibility

### ISSUE THREAD

**Issue by benmiller314** [benmiller314/cdm2019fall#6]
Throughout the semester, I will periodically ask you to develop proposals for a collaborative final project that will allow you to integrate – and extend – what you’ve learned about composing digital media. This is the first of those proposals, and the least formal.

Consider: What further possibilities of sound might you want to explore in connection with other people, and other media?

As always on the forum, replying directly to each other is optional but encouraged: just use the @ symbol to flag someone by username.

**Comment by racheljfu** [comment]
One idea for a group project (that focuses on sound) is a more collaborative soundscape narrative. More specifically, the people in the group could decide a certain location (or multiple locations) for the soundscape to occur in, and then act as different characters in the narrative. Each group member could be a different character and record dialogue / narration, as well as record their own sounds for the narrative. They could share all of this on Github and take turns respectively editing the Audacity project and other files. I haven't thought through all the details yet but this is a rough outline of what could be done collaboratively.

**Comment by fatemaquaid987** [comment]
As @rachelwonder said, I also think that in order to explore different possibilities of sound in connection with other people, the group members can record dialogues as different characters in a story. One of the members can also act as a narrator, narrating the story or different scenes. 
Also, instead of basing our audio projects in one location, one possibility could be to integrate each other's projects to kind of transition from one place to another. 
In connection with other media, I think one possibility could be to make it a video project instead of audio or maybe use animation to visualize it.

**Comment by fathimashabnam** [comment]
I really like @rachelwonder 's and @fatemaquaid987 's suggestions, and would be open to further discussing this. 
Something that interested me a lot with this unit was the idea of our unique and collective perceptions of a particular sound. During the pre-recording stage of my soundscape narrative, I did quite a bit of research on voice acting (partially looking for tips to overcome my dislike of hearing myself recorded, but also because voice acting interests me a lot). As I succumbed to the black hole, that is the YouTube algorithm, I found this specific video, "[The Magic of Making Sound](https://www.youtube.com/watch?v=UO3N_PRIgX0)" that fascinated me (highly recommend watching this, if you have time). 
I think it would be interesting if we could use this concept or recreate this for a group project. Could we build a soundscape (or perhaps even expand to a video accompanied with it) and create different sounds to tell a story instead of recording audio with the video? I think this will push our current projects a little more as it uses sound to enhance the visual experience.

**Comment by emmaknaub** [comment]
I watched the video "The Magic of Making Sound" that @fathimashabnam recommended and it really changed the way I think about sound in videos. I really like the idea of using a concept like that to create a video but with audio that is recorded separately and layered on top of the video. Using this approach would open the door to countless possibilities and would encourage creativity that might not be possible in a video with built-in audio. I'm not sure what exactly we could do for a project, but I really like the idea of using that technique and would be interested in building a project based around that.

**Comment by ram716** [comment]
@emmaknaub ditto

**Comment by epesetsky** [comment]
I decided to give "The Magic of Making Sound" a watch as well and in just the first 60 seconds I am intrigued by this sound making by using other props. It is true when it says "people take sound for granted but you would miss it if it wasn't there". And by recording a more clear and concentrated noise like stepping on a sand bag to replicate the sound of walking in snow, we don't lose that great sound in the midst of recording, but rather enhance it by layering like @emmaknaub mentions. I think this is an interesting idea to explore further, especially if we overlap this with video recording like they do in "The Magic of Making Sound".

**Comment by jlatiniii** [comment]
I also took @fathimashabnam's suggestion of watching "The Magic of Making Sound" and I think this is a really cool idea. I don't know if I would want to do another project just solely based on sound but incorporating this idea of recording sound in a different way could be useful in another project.

**Comment by cjc162** [comment]
I don't really have an idea that comes to mind for exploring sound, but after watching "The Magic of Making Sound" suggested by @fathimashabnam, I agree that something similar could be worth trying out. It's pretty neat how various sounds can be repurposed and I would not mind trying to create something similar.

**Comment by emaur** [comment]
I totally agree with @emmaknaub suggestion of doing a group project where we create the sounds similar to how they do it in the video "The Magic of Making Sound," and then layering it on top of a video. I think it would be both very fun and thought provoking if we then had to guess what each group used to create specific sounds.

**Comment by kevinikea** [comment]
"The Magic of Making Sound" seems like a really cool concept to me.  Totally agree with @emaur about making a video layered with different sounds.  Maybe we could make a soundscape with a video attached to it?  Using sound to make a collage like how people use pictures to one giant mosaic could be something interesting to do, but I'm not quite sure how we would approach that.

**Comment by jadenash21** [comment]
I think Fatima's suggestion would be a cool to do as a group. In middle school, I took a digital media camp where they taught us how sounds in movies are made. It was extremely fascinating to see which objects were used to make certain sounds. I think this would be a cool project to collaborate with others for and see what sounds we are able to master.

**Comment by brittanybenson8** [comment]
I also really like Fatima's suggestion.  I think it would be cool for each group to be assigned a video with no audio, and we have to create the sound.  It is really neat to see what random objects are used to create sound effects.  I think this would be a creatively challenging but fun group project.
